src/App/Pages/Send/SendAddEditPageViewModel.cs

```diff
@@ -200,12 +203,25 @@ public bool ShowPassword
         public bool ShowDeletionCustomPickers => EditMode || DeletionDateTypeSelectedIndex == 6;
         public bool ShowExpirationCustomPickers => EditMode || ExpirationDateTypeSelectedIndex == 7;
         public string ShowPasswordIcon => ShowPassword ? "" : "";
+        public bool DisableHideEmail
```

Comment:
OCD alert: `DisableHideEmail` and `SendOptionsPolicyInEffect` should be above this section (probably right after `ShowPassword` and before `EditMode`), and `DisableHideEmailControl` should be up even higher (probably right after `ShareOnSave` and before `TypeOptions`)